ntbs-service/Models/Notification.cs

```diff
@@ -133,7 +133,7 @@ private string GetNotificationStatusString()
 
         private string FormatDate(DateTime? date)
         {
-            return date?.ToString("dd-MMM-yyyy");
+            return date?.ToString("dd MMM yyyy");
```

Comment:
Could you change all occurrences of this formatting to the new style? OK'ed by Nancy, so it'd be a shame to leave things  inconsistent

Reply:
I did a quick FormatDate search and "dd-MMM-yyyy" search and couldn't find any other occurences - do you know where there might be any others?

Reply:
oh right - i was just surprised we only had so few - well done us! :D